.agents/skills/wpilib-sim/SKILL.md

```
@@ -0,0 +1,218 @@
+   ```
+4. Run normal sim when a human/live tool will drive the robot:
+   ```sh
+   JAVA_HOME=<wpilib-jdk> ./gradlew simulateJava
+   ```
+   For explicit agent-controlled mode selection, pass the JVM property:
+   ```sh
+   JAVA_HOME=<wpilib-jdk> ./gradlew simulateJava -DrobotMode=SIM
+   JAVA_HOME=<wpilib-jdk> ./gradlew simulateJava -DrobotMode=REPLAY
+   ```
+5. Run headless sim when Codex needs to automate checks without opening GUI windows:
+   ```sh
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)